questions

```diff
@@ -0,0 +1 @@
++Billy: Why is this so complicated?
```

Comment:
It's not that complicated.
this is an example of a code-review message on github